flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
what's the justfication? The build failure on aarch64 is fixable

Reply:
plenty of packages build id3 on aarch64.

For example

https://github.com/flathub/org.audacityteam.Audacity/blob/master/deps/libid3tag/libid3tag.json#L42